docs/tutorials/regression.ipynb

```diff
@@ -2,204 +2,391 @@
+       "|  2387 | 2014 |     6 |  10 | … |              0 |          6771 |            1780 | 388525.00000 |\n",
+       "| 15530 | 2015 |     1 |  28 | … |              0 |         11541 |            1260 | 339400.00000 |\n",
+       "|  3605 | 2014 |     6 |  25 | … |              0 |          1202 |            1250 | 345277.77778 |\n",
+       "|  1829 | 2014 |     6 |   2 | … |              0 |          4337 |            2250 | 703114.66667 |\n",
+       "|     … |    … |     … |   … | … |              … |             … |               … |            … |\n",
+       "| 13522 | 2014 |    12 |   5 | … |              0 |          9266 |            2240 | 489377.14286 |\n",
+       "| 16479 | 2015 |     2 |  22 | … |              0 |          6000 |            1400 | 553560.00000 |\n",
+       "| 18536 | 2015 |     3 |  27 | … |              0 |          1380 |            1340 | 293458.33333 |\n",
+       "| 13785 | 2014 |    12 |  10 | … |              0 |          5400 |            1429 | 565200.00000 |\n",
+       "| 20820 | 2015 |     4 |  29 | … |              0 |          2482 |            1290 | 231399.80000 |\n",
+       "+-------+------+-------+-----+---+----------------+---------------+-----------------+--------------+"
+      ]
      },
-     "execution_count": 5,
+     "execution_count": 21,
      "metadata": {},
      "output_type": "execute_result"
     }
+   ],
+   "source": [
+    "prediction = fitted_model.predict(\n",
+    "    test_table\n",
+    ")\n",
+    "# For visualisation purposes we only print out the first 15 rows.\n",
+    "prediction.to_table().slice_rows(start=0, length=15)"
    ]
   },
   {
    "cell_type": "markdown",
-   "source": [
-    "6. You can test the mean absolute error of that model with the initial testing_table as follows:\n"
-   ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "source": [
+    "## Evaluating the Fitted Regressor\n",
+    "\n",
+    "You can test the mean absolute error of that model with the initial testing_table as follows:"
+   ]
   },
   {
    "cell_type": "code",
-   "source": [
-    "test_tabular_dataset = testing_table.to_tabular_dataset(\"price\", extra_names=extra_names)\n",
-    "\n",
-    "fitted_model.mean_absolute_error(test_tabular_dataset)\n"
-   ],
+   "execution_count": 22,
    "metadata": {
-    "collapsed": false,
     "ExecuteTime": {
      "end_time": "2024-05-24T11:02:43.748029100Z",
      "start_time": "2024-05-24T11:02:43.738085900Z"
-    }
+    },
+    "collapsed": false
    },
-   "execution_count": 6,
    "outputs": [
     {
      "data": {
-      "text/plain": "93590.45902700891"
+      "text/plain": [
+       "92672.57153031077"
+      ]
      },
-     "execution_count": 6,
+     "execution_count": 22,
      "metadata": {},
      "output_type": "execute_result"
     }
+   ],
+   "source": [
+    "test_tabular_dataset = testing_table.to_tabular_dataset(\"price\", extra_names=extra_names)\n",
```

Comment:
No need anymore to create a tabular dataset when using metrics. You can just pass the table directly.